Gopkg.lock

```diff
@@ -38,10 +38,10 @@
   version = "v2.0.2"
 
 [[projects]]
-  branch = "master"
+  branch = "add-kvm-port-mappings"
```

Comment:
TODO: change back to master once https://github.com/giantswarm/gsclientgen/pull/25 is merged.